rotors_description/urdf/firefly_generic_odometry_sensor.gazebo

```diff
@@ -26,15 +26,16 @@
 
   <!-- Mount a generic odometry sensor without odometry map (working everywhere). -->
   <xacro:odometry_plugin_macro
-    namespace="${namespace}/odometry_sensor1"
+    namespace="${namespace}"
     odometry_sensor_suffix="1"
     parent_link="${namespace}/base_link"
-    pose_topic="pose"
-    pose_with_covariance_topic="pose_with_covariance"
-    position_topic="position"
-    transform_topic="transform"
-    odometry_topic="odometry"
+    pose_topic="odometry_sensor1/pose"
```

Comment:
Do you want to create a local variable here and then use it in the xacro call (as shown below)?

```XML
<xacro:property name="suffix" value="1" />
<xacro:odometry_plugin_macro
    namespace="${namespace}"
    odometry_sensor_suffix="${suffix}"
    parent_link="${namespace}/base_link"
    pose_topic="odometry_sensor${suffix}/pose"
    ...>
  ...
</xacro:odometry_plugin_macro>
```